src/TaxesApplicator/OrderItemUnitsTaxesApplicator.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\TaxesApplicator;
+
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\Component\Core\Model\TaxRateInterface;
+use Sylius\Component\Core\Taxation\Applicator\OrderTaxesApplicatorInterface;
+use Sylius\Component\Order\Factory\AdjustmentFactoryInterface;
+use Sylius\Component\Taxation\Calculator\CalculatorInterface;
+use Sylius\Component\Taxation\Resolver\TaxRateResolverInterface;
+use Sylius\RefundPlugin\Entity\AdjustmentInterface;
+
+/**
+ * @internal
+ *
+ * This class is not covered by the backward compatibility promise and it will be removed after update Sylius to 1.9.
+ * It is a duplication of a logic from Sylius to provide proper adjustments handling.
+ */
+class OrderItemUnitsTaxesApplicator implements OrderTaxesApplicatorInterface
```

Comment:
Missing final or on purpose? 
```suggestion
final class OrderItemUnitsTaxesApplicator implements OrderTaxesApplicatorInterface
```

Reply:
The same is true for other applicators, but it is true for all the applicators in pure Sylius as well. We could add final tho IMHO